python/ray/_private/accelerators/tpu.py

```diff
@@ -247,3 +191,200 @@ def set_current_process_visible_accelerator_ids(
                 TPU_CHIPS_PER_HOST_BOUNDS_ENV_VAR
             ] = TPU_CHIPS_PER_HOST_BOUNDS_2_CHIP_CONFIG
             os.environ[TPU_HOST_BOUNDS_ENV_VAR] = TPU_SINGLE_HOST_BOUNDS
+        elif num_visible_tpu_chips == 4:
+            os.environ[TPU_CHIPS_PER_HOST_BOUNDS_ENV_VAR] = None
+            os.environ[TPU_HOST_BOUNDS_ENV_VAR] = None
+
+    @staticmethod
+    def _get_tpu_accelerator_type() -> Optional[str]:
```

Comment:
_get_current_node_tpu_accelerator_type

Reply:
actually should it be _get_current_node_tpu_pod_type to avoid the confusion with ray accelerator_type?

Reply:
That makes sense, used `_get_current_node_tpu_pod_type`